server/views/pages/analytics/protectedCharacteristicTemplate.njk

```diff
@@ -87,10 +87,58 @@
   </div>
 
   {% if featureFlags.showAnalyticsPcTrends %}
+
+    {% macro pcGroupDropdown(
```

Comment:
We've been calling these `select` rather than `dropdown` so could name this `pcGroupSelect`

Reply:
FYI: As discussed, I'll rename the other way around: use "dropdown" here but rename related variables.

Reply:
https://github.com/ministryofjustice/hmpps-incentives-ui/pull/194/commits/0eac18a81864fb7d98783d9dc6a15501d8b366b7